packages/mds-geography/tests/index.spec.ts

```diff
@@ -112,100 +105,109 @@ describe('Tests app', () => {
         })
     })
 
+    it('can read a published geography with both read scopes', async () => {
+      await request
+        .get(pathPrefix(`/geographies/${GEOGRAPHY2_UUID}`))
+        .set('Authorization', GEOGRAPHIES_BOTH_READ_SCOPES)
+        .expect(200)
+    })
+
     it('can GET geographies, full version', done => {
-      request
+      void request
         .get(pathPrefix(`/geographies/`))
         .set('Authorization', GEOGRAPHIES_READ_PUBLISHED_SCOPE)
         .expect(200)
         .end((err, result) => {
-          result.body.data.geographies.forEach((item: Geography) => {
-            test.assert(item.geography_json)
-          })
-          test.assert(result.body.version === GEOGRAPHY_API_DEFAULT_VERSION)
-          test.value(result).hasHeader('content-type', APP_JSON)
-          done(err)
+          try {
```

Comment:
Why do we need try/finally if this isn't supposed to throw? Exceptions will cause the test to fail.

Reply:
Exceptions cause it to also time out because done never gets called.

Reply:
If you switch to using `async` you won't have to worry about calling `done()` and you can also remove the `try/finally`

Reply:
Correct